src/mo_frontend/typing.ml

```diff
@@ -1430,9 +1456,9 @@ let resolve_hole env at hole_sort typ =
              match t with
              | T.Obj (T.Module, fs) -> Some (n, fs)
              | _ -> None) |>
-         Seq.filter_map find_candidate |>
+         Seq.filter_map (fun (module_path, fs) -> find_candidate (module_path, fs)) |>
          List.of_seq in
-     Error (List.map (fun candidate -> candidate.desc) lib_candidates)
```

Comment:
There is code similar to this for dot resolutions that may need changing (I basically copied and adapted Christoph's code).